maven/bnd-maven-plugin/README.md

```diff
@@ -137,7 +116,9 @@ The plugin has 3 distinct usage scenarios:
    </plugin>
    ```
 
-2. The second case, let's call it the _filtering_ use case, is a more advanced usage emulating the behavior found in the Bnd _Workspace_ model where instructions are used to include contents in the resulting jar. Note that controlling the actual contents of the jar is handled through configuration of the `maven-jar-plugin`:
+2. The second case, let's call it the _filtering_ case, is a more advanced usage emulating the behavior found in the Bnd _Workspace_ model where instructions are used to include (i.e. *filter*) contents in the jar.
+
+   **Note** that controlling the actual contents of the jar is handled through configuration of the `maven-jar-plugin`:
 
    ```xml
    <plugin>
```

Comment:
It is wrong to show the `<bnd>` example lines having leading spaces. Bnd will interpret this as a line continuation.

Reply:
works perfectly in practice...